internal/network/nsxt_subresource.go

```diff
@@ -0,0 +1,123 @@
+/*
+ *  Copyright 2023 VMware, Inc.
+ *    SPDX-License-Identifier: MPL-2.0
+ */
+
+package network
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	validation_utils "github.com/vmware/terraform-provider-vcf/internal/validation"
+	"github.com/vmware/vcf-sdk-go/models"
+)
+
+// NsxTSchema this helper function extracts the NSX-T schema, which
+// contains the parameters required to install and configure NSX-T in a workload domain.
+func NsxTSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"vip": {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "Virtual IP address which would act as proxy/alias for NSX-T Managers",
```

Comment:
In VCF 5.0, there was a rebranding of "NSX-T" to "NSX". 

All the human readable strings now only refer to "NSX", but the API schema still has "nsxt" for backward compatibility. 

I would suggest the same in the TF code - all human readable strings to be "NSX", "NSX Manager", "NSX Edge" etc.

We could also make the TF resource and member names "nsx" ... but I would leave that up to you ... If you think its better to keep the API member name and schema member name the same, we can leave it how you have now.

Reply:
Converted all NSX-T/NSXT to NSX, except the fields in swagger generated SDK.